shell/README.txt

```diff
@@ -33,5 +33,9 @@ lisa-report  - Pretty format results of last test
 .:: Test commands
 --------------------------------------
 
-lisa-test    - Run tests and assert behaviours
+lisa-test    - Run tests and assert behaviours.
+               This is just a wrapper around exekall that selects all tests
+               modules and use positional arguments as --select patterns.
+               Also the default configuration file will be used if available
```

Comment:
Just to make it absolutely clear to the user - could we mention that the conf file is "target_conf.yml"?